### DISCUSSION TITLE
Missing Kinesis Integration?

### DISCUSSION THREAD

**Discussion by bigbean17** [emqx/emqx#13471] [Q&A] [answered]
Hi, 

I'm trying to connect my deployment to kinesis data streaming. I was following this guide https://docs.emqx.com/en/cloud/latest/data_integration/amazon_kinesis.html, but I couldn't find the `Amazon Kinesis` under the Data Forward category.
<img width="1174" alt="image" src="https://github.com/user-attachments/assets/31ebaf2c-5c3d-4d52-aa9f-2b74a303276f">
Is the integration of Kinesis still available? 

Thanks

**Comment by id** [comment, marked as answer]
Hi @bigbean17. I just created a fresh deployment and I see it right there:
<img width="1442" alt="image" src="https://github.com/user-attachments/assets/82590e68-a8fa-4b6e-ace6-298af5a45b9b">

Please check your deployment version in settings. You need v5 for Kinesis. If you still have v4, you can create a support ticket for the Cloud team to migrate your deployment to v5.
<img width="244" alt="image" src="https://github.com/user-attachments/assets/4f75c405-b680-4cfb-a760-f8fbe96a5d79">

**Reply by bigbean17** [reply]
Thanks, it was the version issue.
